modules/stanford_events_importer/config/optional/ultimate_cron.job.stanford_migrate_stanford_events.yml

```diff
@@ -0,0 +1,17 @@
+langcode: en
```

Comment:
Because stanford_migrate module requires ultimate cron and the importer module require stanford_migrate, i think the ultimate cron job configs can go in `config/install`

Reply:
okay.